src/EditorFeatures/Core/Implementation/NavigateTo/NavigateToIconFactory.cs

```diff
@@ -64,7 +64,7 @@ private static Icon ConvertBitmapSourceToIcon(BitmapSource bmpSource)
 
                     // ICONDIRENTRY: Size. We'll come back to this one once we know what it actually is.
                     offsetOfSize = icoStream.Position;
-                    binaryWriter.Write((int)0);
+                    binaryWriter.Write(0);
```

Comment:
Looks like the (int) cast here was there for clarity with the rest of the casts.